documentation/db/data_dictionary.md

```
@@ -247,16 +300,45 @@
+- **PERMANENTLY_FAILED**: Job has permanently failed after all retry attempts
+
 ---
 
 ## Notes on Data Types
 
+- **Integer**: Whole number (32-bit signed integer, auto-incremented for primary keys)
+- **UInt32**: Unsigned 32-bit integer (ClickHouse specific)
 - **String**: Variable-length character string
-- **Integer**: Whole number (32-bit signed integer)
 - **Decimal (precision, scale)**: Fixed-precision decimal number
     - First number: total number of digits
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)